lib/rake_locales/tasks/locales.rake

```diff
@@ -165,12 +257,21 @@ namespace :locales do
       puts "Sending #{locale} to transifex for translating"
 
       intermediate_file = Rails.root.join('locales', "intermediate_#{locale}.json")
+      mobile_intermediate_file = Rails.root.join('locales/mobile', "intermediate_#{locale}.json")
+
       if File.exist?(intermediate_file)
         locale_db = JSON.parse(File.read(intermediate_file))
 
         transifex_file = Rails.root.join('locales', "transifex_#{locale}.json")
         File.write(transifex_file, JSON.pretty_generate(export_mapper(locale_db)))
       end
+
+      if File.exist?(mobile_intermediate_file)
+        locale_db = JSON.parse(File.read(mobile_intermediate_file))
+
+        transifex_file = Rails.root.join('locales/mobile', "transifex_#{locale}.json")
+        File.write(transifex_file, JSON.pretty_generate(export_mapper(locale_db)))
+      end
       upload_url = "https://www.transifex.com/api/2/project/#{ENV['TRANSLATE_PROJECT']}/resource/for_use_retroelk_transifex_enjson_1_enjson/content/"
 
       cmd = "curl -i -L --user api:#{token} -F file=@#{transifex_file}   -X PUT #{upload_url}"
```

Comment:
This line combined with the new block of code beginning on line 269 would cause us to overwrite all of our english translations for the care portal and the API inside Transifex with just the mobile translations, which is not behavior that we want.